simba/mixins/config_reader.py

```diff
@@ -177,6 +185,9 @@ def __init__(self, config_path: str, read_video_info: bool = True):
             self.p_cols,
             self.clr_lst,
         )
+        self.bodypart_direction = self.read_config_entry(self.config, section=ConfigKey.DIRECTIONALITY_SETTINGS.value,
+                                                         option=ConfigKey.BODYPART_DIRECTION_VALUE.value,
+                                                         data_type=Dtypes.STR.value)
```

Comment:
Have to default to None, so when users created in older versions don't hit an error when path doesnt exist:

        self.bodypart_direction = self.read_config_entry(self.config, section=ConfigKey.DIRECTIONALITY_SETTINGS.value, option=ConfigKey.BODYPART_DIRECTION_VALUE.value, data_type=Dtypes.STR.value, **default_value=Dtypes.NONE.value**)

Reply:
this part is causing bug and has to be resolved